pkg/terraform/exec/plugins/vsphereprivate/internal/archive.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright (c) 2014-2015 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Modified From:
+// https://github.com/vmware/govmomi/blob/e651475773eba5139c706613b3b035377d6c1c30/govc/importx/archive.go
```

Comment:
would be very nice if we can list high level changes from the source..

Reply:
@abhinavdahiya I reviewed what we were using archive for and we didn't need to copy it in - so I removed it.